src/main.scss

```
@@ -99,6 +99,23 @@ body {
   }
 }
 
+// Truncated element in a flexbox child must not fully collapse when the
+// flex container is resized down. 
+// @see https: //drafts.csswg.org/css-flexbox/#min-size-auto
```

```suggestion
// @see https://drafts.csswg.org/css-flexbox/#min-size-auto
```
what was the behavior before?